packages/aws-cdk-lib/recommended-feature-flags.json

```diff
@@ -78,5 +78,6 @@
   "@aws-cdk/s3-notifications:addS3TrustKeyPolicyForSnsSubscriptions": true,
   "@aws-cdk/aws-ec2:requirePrivateSubnetsForEgressOnlyInternetGateway": true,
   "@aws-cdk/aws-s3:publicAccessBlockedByDefault": true,
+  "@aws-cdk/aws-s3-deployment:defaultMemoryLimit": true,
```

Comment:
@aws-cdk/aws-s3-deployment:defaultMemoryLimit

this is not descriptive for me. How about rename to 

@aws-cdk/aws-s3-deployment:default512MemoryLimit ?

And update all occurrences?